server/automute_channel.go

```diff
@@ -45,6 +45,12 @@ func (p *Plugin) updateAutomutingOnChannelCreated(channel *model.Channel) error
 		return nil
 	}
 
+	canAutomute, err := p.canAutomuteChannel(channel)
```

Comment:
Can we remove the `channel.IsGroupOrDirect` above now that it's part of the `canAutomuteChannel` check?

Reply:
We could, but it still servers a purpose to short-circuits channels and not even bother with canAutomuteChannel.